Dalamud/Interface/Internal/Windows/Settings/SettingsWindow.cs

```diff
@@ -157,6 +157,8 @@ public override void Draw()
             }
         }
 
+        ImGui.EndTabBar();
```

Comment:
Should be moved 2 lines up due to ending a conditional check, where this can break.

Reply:
Oh good catch, I believe I moved it to the correct location now.